contracts/package.json

```diff
@@ -94,6 +94,7 @@
     "test": "yarn test:hardhat && yarn test:forge",
     "test:forge": "forge test",
     "test:hardhat": "hardhat test",
-    "test:gas": "forge test --match-path test/foundry/PerformanceTesting.t.sol --gas-report"
+    "test:gas": "forge test --match-path test/foundry/PerformanceTesting.t.sol --gas-report",
+    "hardhat": "hardhat"
```

Comment:
Note: this was necessary because we calling `hardhat` outside of yarn caused an issue with hardhat believing it wasn't in the correct directory.